2023-07-30 14:22:05.138 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-07-30 14:22:05.138 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration snooz which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-07-30 14:22:05.138 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration dreo which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-07-30 14:22:08.299 ERROR (MainThread) [custom_components.hacs] HACS is disabled - invalid_token
2023-07-30 14:22:08.303 ERROR (MainThread) [custom_components.hacs] Bad credentials
Traceback (most recent call last):
  File "/config/custom_components/hacs/base.py", line 464, in async_can_update
    response = await self.async_github_api_method(self.githubapi.rate_limit)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hacs/base.py", line 522, in async_github_api_method
    raise HacsException(_exception)
custom_components.hacs.exceptions.HacsException: Bad credentials
2023-07-30 14:22:10.969 ERROR (SyncWorker_3) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:22:14.041 ERROR (SyncWorker_3) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:22:17.113 ERROR (SyncWorker_3) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:22:17.115 ERROR (SyncWorker_3) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:22:17.116 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Roomba' for roomba integration not ready yet; Retrying in background
2023-07-30 14:22:20.185 ERROR (SyncWorker_3) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:22:23.257 ERROR (SyncWorker_3) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:22:26.329 ERROR (SyncWorker_3) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:22:26.331 ERROR (SyncWorker_3) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:22:39.833 ERROR (SyncWorker_6) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:22:42.905 ERROR (SyncWorker_6) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:22:45.977 ERROR (SyncWorker_6) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:22:45.978 ERROR (SyncWorker_6) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:23:09.146 ERROR (SyncWorker_1) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:23:12.217 ERROR (SyncWorker_1) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:23:15.289 ERROR (SyncWorker_1) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:23:15.290 ERROR (SyncWorker_1) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:23:58.841 ERROR (SyncWorker_4) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:24:01.913 ERROR (SyncWorker_4) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:24:04.985 ERROR (SyncWorker_4) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:24:04.986 ERROR (SyncWorker_4) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:25:28.249 ERROR (SyncWorker_9) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:25:31.321 ERROR (SyncWorker_9) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:25:34.393 ERROR (SyncWorker_9) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:25:34.394 ERROR (SyncWorker_9) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:26:57.913 ERROR (SyncWorker_8) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:27:00.986 ERROR (SyncWorker_8) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:27:04.057 ERROR (SyncWorker_8) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:27:04.059 ERROR (SyncWorker_8) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:28:27.481 ERROR (SyncWorker_2) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:28:30.553 ERROR (SyncWorker_2) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:28:33.625 ERROR (SyncWorker_2) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:28:33.626 ERROR (SyncWorker_2) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:29:56.953 ERROR (SyncWorker_9) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:30:00.025 ERROR (SyncWorker_9) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:30:03.097 ERROR (SyncWorker_9) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:30:03.098 ERROR (SyncWorker_9) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:31:26.425 ERROR (SyncWorker_8) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:31:29.497 ERROR (SyncWorker_8) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:31:32.569 ERROR (SyncWorker_8) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:31:32.571 ERROR (SyncWorker_8) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:32:55.993 ERROR (SyncWorker_2) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:32:59.065 ERROR (SyncWorker_2) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:33:02.137 ERROR (SyncWorker_2) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:33:02.139 ERROR (SyncWorker_2) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:34:25.305 ERROR (SyncWorker_6) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:34:28.378 ERROR (SyncWorker_6) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:34:31.449 ERROR (SyncWorker_6) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:34:31.450 ERROR (SyncWorker_6) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:35:54.777 ERROR (SyncWorker_3) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:35:57.849 ERROR (SyncWorker_3) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:36:00.921 ERROR (SyncWorker_3) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
2023-07-30 14:36:00.922 ERROR (SyncWorker_3) [roombapy.remote_client] Unable to connect to 192.168.0.212
2023-07-30 14:37:24.441 ERROR (SyncWorker_4) [roombapy.remote_client] Can't connect to 192.168.0.212, error: [Errno 113] Host is unreachable
